src/wymeditor/editor/base.js

```diff
@@ -442,15 +442,24 @@ WYMeditor.editor.prototype._docEventQuirks = function () {
 */
 WYMeditor.editor.prototype._bindUIEvents = function () {
     var wym = this,
+        $toolbarButtons = jQuery(wym._box).find(wym._options.toolSelector),
+        dialogButtonSelector = wym._options.dialogButtonSelector ||
+            WYMeditor.DIALOG_BUTTON_SELECTOR,
         $html_val;
 
-    // Tools buttons
-    jQuery(wym._box).find(wym._options.toolSelector).click(function () {
+    // Action buttons
+    $toolbarButtons.not(dialogButtonSelector).click(function () {
```

Comment:
I agree, completely. The `.exec` thing is confusing, especially when mistaken for something to do with `execCommand`.